### ISSUE TITLE
NTR: Skin biopsy: granuloma (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9322]
New term request Skin biopsy: granuloma (ang_130518232528):   UMLS: 0151684

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000070
- label: Cutaneous granuloma
- parents: [Granuloma (HP:0032252)](https://hpo.jax.org/app/browse/term/HP:0032252)
- pmid: PMID:36902053
- def: A granuloma localized to the skin, that is, a chronic inflammatory manifestation with localized aggregation of histiocytes with or without other inflammatory cells (such as plasma cells, eosinophils, or neutrophils), with or without necrosis, with or without vasculitis, with or without calcification, and with or without foreign bodies. Granulomas may be due to infection or chronic inflammatory disease or reactions to foreign material.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9322
- synonyms:
